packages/appium/docs/en/ecosystem/index.md

```diff
@@ -53,3 +56,15 @@ the Appium client depends on) since that is what you will use as your primary in
 
 In general, any W3C WebDriver spec-compatible client will also integrate well with Appium, though
 some Appium-specific commands may not be implemented in other clients.
+
+## Plugins
```

Comment:
not sure if we need to duplicate these lists here as we already have them in readme

Reply:
this is for the documentation. the readme won't be part of the documentation. when we publish the docs, we should link to these pages in the readme instead.

Reply:
my point is we should not have duplicates